app/routes/documentation.js

```diff
@@ -21,7 +21,7 @@ import {renderHtml, renderMarkdown} from '../../lib/renderers.js';
 
 const router = express.Router();
 
-router.use(express.static('./public'));
+// router.use(express.static('./public'));
```

Comment:
@ed-bird Yeah. I can just remove it for clarity. Static file rendering is now done through the `staticFileRenderer` rather than express's built-in static handler, so as to enable the env var stuff. I just commented this out to show the change explicitly, but can remove it all the same.